src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -102,48 +101,55 @@ def generate_MCP_1_default_config(self, filename):
     pal_low_frequency = 3550000000.0
     pal_high_frequency = 3560000000.0        
 
-    ppa_record_1, pal_record_1 = makePpaAndPalRecordsConsistent(ppa_record,
+    ppa_records_1, pal_records_1 = makePpaAndPalRecordsConsistent(ppa_record,
                                                                 [pal_record],
                                                                 pal_low_frequency,
                                                                 pal_high_frequency,
                                                                 'test_user_1')
     # Load DPA record
-    dpa_1 = json.load(
-      open(os.path.join('testcases', 'testdata', 'dpa_0.json')))
-    dpa_2 = json.load(
-      open(os.path.join('testcases', 'testdata', 'dpa_1.json')))
-    dpa_3 = json.load(
-      open(os.path.join('testcases', 'testdata', 'dpa_0.json')))
+    dpa_1 = {
+       'dpaId': 'test_dpa_id1',
```

Comment:
These should be real DPA IDs.

Reply:
Modified   the DPA Ids per NTIA definition.